app/src/organisms/ProtocolsLanding/ProtocolsEmptyState.tsx

```diff
@@ -19,10 +20,14 @@ export function ProtocolsEmptyState(): JSX.Element | null {
       width="100%"
       padding={SPACING.spacing4}
       paddingTop={SPACING.spacing6}
+      position={POSITION_ABSOLUTE}
+      top="50%"
+      left="50%"
+      transform="translateY(25%) translateX(-50%)"
```

Comment:
This feels a bit suspicious to me. Can't you achieve the same thing by only adding a translateY? 

```suggestion
      transform="translateY(25%)"
```

Reply:
Thanks! You are right.